src/statement.js

```diff
@@ -518,61 +518,73 @@ pp.parseClass = function(node, isStatement) {
   this.expect(tt.braceL)
   while (!this.eat(tt.braceR)) {
     if (this.eat(tt.semi)) continue
```

Comment:
Since this is only optional for methods, maybe it should go into `parseClassMember`.

Reply:
Semicolon is an empty `ClassElement`: https://www.ecma-international.org/ecma-262/8.0/#prod-ClassElement
So I think it should in `parseClassMember`.

Reply:
I think it's alright if `parseClassMember` returns `null`.